charts/keep/README.md

```diff
@@ -97,27 +97,42 @@ Keep Helm Chart
 | backend.waitForDatabase.port | int | `3306` |  |
 | database.affinity | object | `{}` |  |
 | database.autoscaling.enabled | bool | `false` |  |
+| database.cleanup.days | int | `90` |  |
+| database.cleanup.enabled | bool | `false` |  |
+| database.cleanup.schedule | string | `"0 1 * * *"` |  |
+| database.config | object | `{}` |  |
 | database.enabled | bool | `true` |  |
-| database.env[0].name | string | `"MYSQL_ALLOW_EMPTY_PASSWORD"` |  |
-| database.env[0].value | bool | `true` |  |
-| database.env[1].name | string | `"MYSQL_DATABASE"` |  |
-| database.env[1].value | string | `"keep"` |  |
-| database.env[2].name | string | `"MYSQL_PASSWORD"` |  |
-| database.env[2].value | string | `nil` |  |
 | database.extraVolumeMounts | list | `[]` |  |
 | database.extraVolumes | list | `[]` |  |
 | database.healthCheck.enabled | bool | `false` |  |
-| database.healthCheck.probes.livenessProbe.tcpSocket.port | int | `3306` |  |
-| database.healthCheck.probes.readinessProbe.initialDelaySeconds | int | `30` |  |
-| database.healthCheck.probes.readinessProbe.periodSeconds | int | `10` |  |
-| database.healthCheck.probes.readinessProbe.tcpSocket.port | int | `3306` |  |
+| database.healthCheck.probes | object | `{}` |  |
 | database.image.pullPolicy | string | `"IfNotPresent"` |  |
-| database.image.repository | string | `"mysql"` |  |
+| database.image.repository | string | `""` |  |
 | database.image.tag | string | `"latest"` |  |
 | database.imagePullSecrets | list | `[]` |  |
+| database.mysql.dataDir | string | `"/var/lib/mysql"` |  |
+| database.mysql.env[0].name | string | `"MYSQL_ALLOW_EMPTY_PASSWORD"` |  |
+| database.mysql.env[0].value | bool | `true` |  |
```

Comment:
Is it good that it’s yes be default?

Reply:
Technically not, but thats how it was. I can change the default value here, but would It make sense to add then a `CHANGELOG`? Otherwise that might cause some trouble if you blindly upgrade? And I would then maybe prefer to do it inside a separate PR.

Reply:
Lets leave it as is for now. I'm merging :)